index.html

```diff
@@ -0,0 +1,86 @@
+<!DOCTYPE html>
+<html>
+
+  <nav id="navbar">
+  <header>User's Manual</header>
+ 
+    
+    <ul>
+      <li><a class="nav-link" href="#Setting_up_your_oven"> Setting up your oven </a></li>
+      <li><a class="nav-link" href="#Clock_Setting"> Clock Setting </a></li>
+      <li><a class="nav-link" href="#Operation"> Operation </a></li>
+      <li><a class="nav-link" href="#Maintenance"> Maintenance </a></li>
+      <li><a class="nav-link" href="#Warranty_information"> Warranty information </a></li>
+       </nav>
+
+<head>
+  <body>
+  <h1>Microwave user's manual for Dummies</h2>
```

Comment:
Be careful with mixing `<hx>` tags. You can choose only one. In this particular line, you could choose `<h1>` since it will define the most important header (which is the title of your Manual).